specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-04-01/machineLearningServices.json

```diff
@@ -1004,6 +1004,61 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/customServices": {
```

Comment:
This needs to be an action word like createCustomServices or something similar to indicate that this is an action being done on the compute

Reply:
from the usage it seems like updateCustomServices might be more appropriate

Reply:
this has always been just customServices even in preview. Its because we mention the action as a part of REST verb i.e POST. 

As per guidelines, "customServices" are our resource collection: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#:~:text=ex%3A%20blobcontainer%2C%20myqueue)-,resource%E2%80%91collection,-Name%20of%20the

Reply:
You can't perform Action on a Resource Type. Action is performed on an instance of a Resource Type, so it will either be:

POST "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/updateCustomServices"
// which means Action "updateCustomServices" is performed on instance "computeName" of Resource Type "Microsoft.MachineLearningServices/workspaces/computes"

or

POST "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/customServices/customService1/update"
// which means Action "update" is performed on instance "customService1" of Resource Type "Microsoft.MachineLearningServices/workspaces/computes/customServices"

Reply:
Discussed this with team and decided that we don't need to publish this API at all. We can leverage PUT /computes to update the custom services